### PR TITLE
Make talks more awesome

### PR DESCRIPTION
**Pull request by sentientmonkey**
Based off of @futureperfect PR #145 

Only shows 6 more recent completed talks now on talks page - linking to the past talks to see all. Also breaking out talk proposal to it's own page to get more screen real estate. Had to change up some of the existing tests to get this all working happy. Might still need a bit of cleanup 🤷‍♂️

### CODE REVIEW

**Comment by futureperfect** [comment]
I surfed Scott’s shoulder last week while he worked on this. We should favor this PR over #145

File: app/controllers/talks_controller.rb [hunk 1]
```diff
@@ -42,6 +46,10 @@ def talks
     @talks ||= Talk.available
   end
 
+  def recently_given
+    @recent_talks ||= Talk.recently_given
```
Comment (phiggins) [inline app/controllers/talks_controller.rb]: nitpick: Please name the method and instance variable the same.

Comment (phiggins) [reply app/controllers/talks_controller.rb]: To clarify, I mean the instance variable and controller method should be the same, the model method is fine.

**Comment by phiggins** [comment]
I'd prefer to remove the suggestions table and model as well, because we're unlikely to do anything with that data and it will likely just add clutter to the project.

**Comment by phiggins** [comment]
Thanks for the updates!

**Pull request closed without merging**